work-in-progress/WIP-json-schema-release-notes.md

```diff
@@ -0,0 +1,52 @@
+---
+title: JSON Schema Draft-07 Release Notes
+layout: page
+---
+
+There are a few breaking changes with newer keywords based on community feedback,
+and internally there has been a lot of restructuring through a new concept called
+vocabularies, but on the whole things are mostly the same.
+
+* TOC
+{:toc}
+
+- Update to RFC 8359 for JSON specification
+- Add the concept of formal vocabularies, and how they can be recognized through meta-schemas
+- Formalized annotation collection
+- Moved applicator keywords from the Validation specification as their own vocabulary
+- Define "$ref" behavior in terms of the assertion, applicator, and annotation model
+- Note undefined behavior for "$ref" targets involving unknown keywords
+- Additional guidance on initial base URIs beyond network retrieval
+- Allow "schema" media type parameter for "application/schema+json"
+
+### Keywords
+
+* Eight brand new keywords were added
+* One keyword was renamed
+* One keyword was split in half
+* One keyword in Core changed behavior
+* One keyword in Hyper-Schema changed behavior
+
+keyword | change | notes
+---- | ---- | ----
+[`"definitions"`](json-schema-core.html#rfc.section.TODO) | **removed** | use new "$defs" core keyword
+[`"unevaluatedProperties" and "unevaluatedItems"`](json-schema-core.html#rfc.section.TODO) | **added** |
+[`"$ref"`](json-schema-core.html#rfc.section.TODO) | **changed** | can now have siblings (keywords next to it)
+[`"$defs"`](json-schema-core.html#rfc.section.10.TODO) | **added** | moved over to core from validation
+`"dependencies"` | **removed** | use "dependentRequired" or "dependentSchemas"
+[`"dependentRequired"`](json-schema-core.html#rfc.section.10.TODO) | **added** | added to core schema
+[`"dependentSchemas"`](json-schema-core.html#rfc.section.10.TODO) | **added** | added to core schema
+[`"applicator"`](json-schema-core.html#rfc.section.10.TODO) | **moved** | from validation to core, but is functionally identical
```

Comment:
`applicator` is not a keyword, and wasn't even a section in the old draft.  This needs to be presented separately from a table of keywords.